scm/src/GFS_typedefs.F90

```diff
@@ -132,20 +128,19 @@ module GFS_typedefs
 
     character(len=32), pointer :: tracer_names(:) !< tracers names to dereference tracer id
                                                   !< based on name location in array
-    character(len=64) :: fn_nml                   !< namelist filename
+    character(len=65) :: fn_nml                   !< namelist filename
```

Comment:
I forgot about this one. I don't think it matters, but Phil Pegion fixed this to be consistent with the rest of the file for SCM. I don't know why it is length 65 in this one place.